App/notReadyTests/RegularGroups.test.tsx

```diff
@@ -0,0 +1,24 @@
+// import { render } from "@testing-library/react";
+// import DisplayGroups from "../src/pages/RegularGroups";
+// import React from "react";
+// import { ChakraProvider } from "@chakra-ui/react";
+// import theme from "../theme";
+
+// describe("Add User to Group Test Case", () => {
```

Comment:
Why is this test commented out?

Reply:
Not able to work on time, will fix next sprint